app/src/main/java/com/yagni/model/ReadOneOrder.java

```diff
@@ -47,10 +47,10 @@ public void readOneOrder(DbConnection linkDb){
             productQuantity = result.getString(6);
             String [] row = {orderNum, orderDate, hashEmail, location, productID, productQuantity};
             tableModel.addRow(row);
-            
+
```

Comment:
delete the whitespace between your catch exception and your closing curly on the while statement.